src/Alt.ts

```diff
@@ -32,6 +32,13 @@ export interface Alt2<F extends URIS2> extends Functor2<F> {
   readonly alt: <E, A>(fx: Kind2<F, E, A>, fy: () => Kind2<F, E, A>) => Kind2<F, E, A>
 }
 
+/**
+ * @since 2.0.0
```

Comment:
should be `@since 2.2.0` (same for the other 3C interfaces..)

Reply:
Ok it will then be 2.2.0 :)